metanome-cli/README.md

```diff
@@ -0,0 +1,40 @@
+# Metanome CLI
+
+The main purpose of this project is to provide a command-line interface for [Metanome](https://github.com/HPI-Information-Systems/Metanome) to allow for easy automation of data profiling tasks, e.g., to conduct experiments or to profile datasets batchwise.
+Besides that, this project integrates Metanome with
+* [Metacrate](https://github.com/stratosphere/metadata-ms), a storage and analytics tool for data profiles, and
+* [ProfileDB](https://github.com/sekruse/profiledb-java), a tiny tool to collect and store experimental data.
+
+Furthermore, HDFS is supported as input source.
+
+## Installation
+
+The project can be built with Maven:
+```
+.../metanome-cli$ mvn package
+```
+This command creates a "fatjar" (`target/metanome-cli-0.1-SNAPSHOT.jar` or similar) that contains Metanome and the Metanome CLI along with all their dependencies (except for Metanome algorithms, though).
+
+Note that this project might depend on unstable snapshot versions of [Metanome](https://github.com/HPI-Information-Systems/Metanome), [Metacrate](https://github.com/stratosphere/metadata-ms), and [ProfileDB](https://github.com/sekruse/profiledb-java).
```

Comment:
Unfortunately, this is still true. We could think about removing these altogether, as they are not strictly necessary for CLI operation.

Reply:
I removed this now.